src/cf-tables.less

```diff
@@ -370,11 +380,24 @@
 }
 
 .table__sortable {
+    button.sortable {
+        width: 100%;
+        height: 100%;
+        padding: inherit;
+        margin: inherit;
+        border: none;
+        background: none;
+        font-family: inherit;
+        font-weight: inherit;
+        outline: none;
+        text-align: left;
+    }
```

Comment:
Yeah, it's a good point. I thought about that but decided not to mention it.

Here's an idea: What if we created a ".u-unstyled-button" utility class/mixin in cf-core and used that both here and as a base for `.btn__link`? I don't think I'd hold up this PR while waiting for that to happen, but tell me if you think that's a good idea.

/cc @ascott1 @jimmynotjim @KimberlyMunoz